### PR TITLE
Add statistical stopping test

### PR DESCRIPTION
**Pull request by odow**

### CODE REVIEW

File: src/plugins/stopping_rules.jl [hunk 1]
```diff
@@ -58,6 +23,11 @@ end
 
 # ========================= Time Limit Stopping Rule ========================= #
 
+"""
+    TimeLimit(limit::Float64)
+
+Teriminate the algorithm after `limit` seconds of computation.
```
Comment (frapac) [inline src/plugins/stopping_rules.jl]: Typos!

File: src/plugins/stopping_rules.jl [hunk 2]
```diff
@@ -70,17 +40,46 @@ end
...
+    current_bound = log[end].bound
+    return sample_mean - sample_ci  <= current_bound <= sample_mean + sample_ci
+end
 
 # ======================= Bound-stalling Stopping Rule ======================= #
 
+"""
+    BoundStalling(num_previous_iterations::Int, tolerance::Float64)
+
+Teriminate the algorithm once the deterministic bound (lower if minimizing,
```
Comment (frapac) [inline src/plugins/stopping_rules.jl]: Idem

File: src/Kokako.jl [hunk 3]
```diff
@@ -17,18 +17,21 @@ export @stageobjective
 # Modelling interface.
 include("user_interface.jl")
 
-# SDDP related modular utilities.
-include("plugins/risk_measures.jl")
-include("plugins/sampling_schemes.jl")
-include("plugins/bellman_functions.jl")
-include("plugins/stopping_rules.jl")
+# Default definitions for SDDP related modular utilities.
+include("plugins/headers.jl")
```
Comment (frapac) [inline src/Kokako.jl]: I like that

File: src/plugins/headers.jl [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+
+"""
+    adjust_probability(measure::Expectation
+                       risk_adjusted_probability::Vector{Float64},
+                       original_probability::Vector{Float64},
+                       noise_support::Vector{Noise{T}},
+                       objective_realizations::Vector{Float64},
+                       is_minimization::Bool) where T
+"""
+function adjust_probability end
```
Comment (frapac) [inline src/plugins/headers.jl]: Do you plan to use quantization to reduce noises' support sizes?

File: src/plugins/headers.jl [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+    error("Kokako.refine_bellman_function not implemented for " *
+          "$(bellman_function).")
+end
+
+"""
+    bellman_term(::AbstractBellmanFunction)
+
+Return a JuMP expression representing the Bellman function.
+"""
+function bellman_term(bellman::AbstractBellmanFunction)
```
Comment (frapac) [inline src/plugins/headers.jl]: I think the name of this function could me more explicit

File: src/plugins/headers.jl (hunk 4 above)
Comment (odow) [reply src/plugins/headers.jl]: Care to elaborate on this? What do you mean?

Comment (frapac) [reply src/plugins/headers.jl]: You currently use this function to build probability weights adjusted to a risk measure. I was wondering if we could also adjust the probability weights to decrease the size of the discrete support so as to improve numerical efficiency. You may have read this idea in [this article, p.3](https://pdfs.semanticscholar.org/6a29/e7bf2ffbe3aa010eb097e57460b5b06f10c8.pdf). 
This [other](https://arxiv.org/pdf/1701.04072.pdf) article (section 4, p.18) details also some good ideas but I didn't test them.

**Pull request merged**